tests/algorithms/test_gradient_clipping.py

```diff
@@ -170,10 +221,16 @@ def test_gradient_clipping_algorithm_with_fsdp_enabled(
     engine.run_event(Event.AFTER_TRAIN_BATCH)
```

Comment:
Renamed to test_gradient_clipping_algorithm_with_fsdp_enabled_does_not_error, since the other tests do a decent job testing functionality. Thanks for suggestion.